release-notes/2023-10/README.md

```diff
@@ -0,0 +1,244 @@
+
+You can specify the username and password in the global (`mongodb+srv://user:pass@host/<database>`) `<database>` will be replaced during per-service Helm install/upgrade. This forces SystemLink Enterprise to use the same username and password for all databases hosted in your MongoDB instance.
+
+You can also use per-service username and password combinations (`mongodb+srv://<username>:<password>@host/<database>`) `<username>`, `<password>`, and `<database>` will be replaced during per-service Helm install/upgrade. This forces SystemLink Enterprise to use your specified usernames and passwords for each database hosted in your MongoDB instance.
+
+### Data Frame Service
+
+`dataframeservice.requestBodySizeLimitMegabytes` has been renamed to `dataframeservice.requestBodySizeLimit`. It now accepts units in "MiB" (Mebibytes, 1024 KiB) or in "MB" (Megabytes, 1000 KB).
+    - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-10/getting-started/templates/systemlink-values.yaml#L579)
+
+### Tag Historian service
+
+The Tag Historian service is included in the SystemLink Enterprise top level Helm chart.
+    - You must configure the secrets for MongoDB required by this service.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-10/getting-started/templates/systemlink-secrets.yaml#L549)
+    - You must also configure a `continuationTokenEncryptionKey`. When creating the `continuationTokenEncryptionKey`, use a 32-byte cryptographically random value which is base64 encoded.
+        - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2023-10/getting-started/templates/systemlink-secrets.yaml#L566)
+
+## Upgrade Considerations
+
+- DataFrame Service Kafka dependency has been removed
+    - The DataFrame Service now uses a more efficient method for writing data to new tables, replacing Kafka. The DataFrame Service will still use Kafka for data ingestion for tables created before the 2023-10 release, while tables created after upgrading to the 2023-10 release will have data written directly to S3 storage. This changes leads to greatly reduced resource utilization.
+    - After upgrading to the 2023-10 release, you can safely remove Kafka from your cluster once all pre-upgrade tables are set to readonly. Please note that disabling Kafka may lead to data loss if pre-upgrade tables are not readonly, because any buffered data may not get written to storage.
+    - Refer to [Remove-Kafka-from-the-cluster.md](https://github.com/ni/install-systemlink-enterprise/tree/2023-10/release-notes/2023-10/Remove-Kafka-from-the-cluster.md) for detailed instructions.
+- DataFrame Service Dremio refresh interval
+    - The Dremio data set refresh job interval was increased from 2 minutes to 1 hour. This reduces overall load on Dremio.
+    - Customers are not required to uptake this change, but doing so will result in lower load and lower resource utilization.
```

Comment:
Customers are required to uptake this change. This change prevents Dremio's volumes from filling up, which can get it into a bad state.

Reply:
Updated based on input